epictrack-web/src/components/myWorkplans/Card/CardBody.tsx

```diff
@@ -15,7 +15,7 @@ const IndicatorSmallIcon: React.FC<IconProps> = Icons["IndicatorSmallIcon"];
 const ClockIcon: React.FC<IconProps> = Icons["ClockIcon"];
 
 const CardBody = ({ workplan }: CardProps) => {
-  const phase_color = workplan?.phase_info?.work_phase?.phase?.color;
+  const phase_color = "#003366";
```

Comment:
Please search this color in the Theme.tsx file and use the variable here

Reply:
changed its slightly different but i changed it to main